Registry/Registry.NMM

```diff
@@ -871,7 +871,13 @@ state   real    qnwfa          ikjftb  scalar      1         m     i01h023ru=(Up
 state   real    qnifa          ikjftb  scalar      1         m     i01h023ru=(UpMassIKJ:@ECopy,0.0)d=(DownMassIKJ:@ECopy,0.0)f=(BdyMassIKJ:@ECopy,0.0)       "QNIFA"            "ice-friendly aerosol number con"   "# kg(-1)"
 state   real    qndrop         ikjftb  scalar      1         m     i01h023ru=(UpMassIKJ:@ECopy,0.0)d=(DownMassIKJ:@ECopy,0.0)f=(BdyMassIKJ:@ECopy,0.0)       "QNDROP"        "Droplet number mixing ratio"        "# kg-1"
 state   real    qrimef         ijkftb  scalar      1         m     i01h023ru=(UpMass:@ECopy,0.0)d=(DownMass:@ECopy,0.0)f=(BdyMass:@ECopy,0.0)       "QRIMEF"           "rime factor * qi" "kg kg-1"
-
+#---------------------------------ntu3m-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+state   real    qdcn           ijkftb  scalar      1         m     i01h023ru=(UpMass:@ECopy,0.0)d=(DownMass:@ECopy,0.0)f=(BdyMass:@ECopy,0.0)       "QDCN"     "Mass mixing ratio for dry CN" "kg kg(-1)"
+state   real    qtcn           ijkftb  scalar      1         m     i01h023ru=(UpMass:@ECopy,0.0)d=(DownMass:@ECopy,0.0)f=(BdyMass:@ECopy,0.0)       "QTCN"     "Mass mixing ratio for total CN" "kg kg(-1)"
+state   real    qccn           ijkftb  scalar      1         m     i01h023ru=(UpMass:@ECopy,0.0)d=(DownMass:@ECopy,0.0)f=(BdyMass:@ECopy,0.0)       "QCCN"     "Mass mixing ratio for cloud CN" "kg kg(-1)"
+state   real    qrcn           ijkftb  scalar      1         m     i01h023ru=(UpMass:@ECopy,0.0)d=(DownMass:@ECopy,0.0)f=(BdyMass:@ECopy,0.0)       "QRCN"     "Mass mixing ratio for rain CN" "kg kg(-1)"
+state   real    qnin           ijkftb  scalar      1         m     i01h023ru=(UpMass:@ECopy,0.0)d=(DownMass:@ECopy,0.0)f=(BdyMass:@ECopy,0.0)       "QNIN"     "Potential ice nuclei number mixing ratio" "# kg(-1)"
+#---------------------------------ntu3m--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
@tzuchin12 These are probably needed for the NMM registry.

Reply:
I can not catch the point. They are already listed in Registry.NMM.

Reply:
@tzuchin12 I meant new variables qdcn, qtcn, qccn, qrcn and qnin.

Reply:
These five new variables are required to NTU_init through the usage of module_state_description in module_mp_ntu.F.

Reply:
@tzuchin12 I don't think these are needed. For example, if you check a variable used by P3 scheme, vmi3d, you will see that this variable, together with some others unique to P3 are not declared in NMM registry.

Reply:
Some modifications were made as suggested.